engine/src/cython/engine.cpp

```diff
@@ -65,22 +77,39 @@ std::pair<std::vector<ral::io::data_loader>, std::vector<ral::io::Schema>> get_l
 			parser = std::make_shared<ral::io::csv_parser>(args_map);
 		} else if(fileType == ral::io::DataType::ARROW){
 	     	parser = std::make_shared<ral::io::arrow_parser>(tableSchema.arrow_table);
-		}
+		} else if(fileType == ral::io::DataType::MYSQL) {
+#ifdef MYSQL_SUPPORT
+      parser = std::make_shared<ral::io::mysql_parser>();
+      auto sql = ral::io::getSqlInfo(args_map);
+      provider = std::make_shared<ral::io::mysql_data_provider>(sql);
+      isSqlProvider = true;
+#endif
```

Comment:
there should be an else to the `#ifdef MYSQL_SUPPORT` so that if its not compiled with that support and someone tries to make a table with that, that is outputs an appropriate error

Reply:
Done